modelplugin/aether-1.0.0/yang/aether-profile@2020-07-15.yang

```diff
@@ -0,0 +1,52 @@
+module aether-connectivity-profile {
+  namespace "http://opennetworking.org/aether/connectivity-profile";
+  prefix pro;
+
+  organization "Open Networking Foundation.";
+  contact "Scott Baker";
+  description "To generate JSON from this use command
+    pyang -f jtoxx test1.yang | python3 -m json.tool > test1.json
+    Copied from YangUIComponents project";
+
+  revision "2020-07-15" {
+    description "An Aether Connectivity Profile";
+    reference "RFC 6087";
+  }
+
+  typedef proid {
```

Comment:
If you import ietf-yang-types.yang (copy it from the "Devicesim" folder), you will find a type `uuid` there that you can use instead of defining a new one

See https://github.com/onosproject/config-models/blob/e7b51266a2fd74605ac9d4bf0febe16e0187a9d0/modelplugin/devicesim-1.0.0/yang/ietf-yang-types%402013-07-15.yang#L445